ui-ngx/src/app/modules/home/components/profile/tenant-profile.component.ts

```diff
@@ -76,6 +76,52 @@ export class TenantProfileComponent extends EntityComponent<TenantProfile> {
         additionalInfo: {
           description: ''
         }
+      },
+      {
+        id: guid(),
+        name: 'HighPriority',
+        topic: 'tb_rule_engine.hp',
+        pollInterval: 25,
```

Comment:
I would like to set pool interval as lazy as possible.

Even 1000ms for 500 tenants*2partitions*(main,hp,sq)3queues = 3000 poll per second for the cluster. Please estimate the load before provide such a parameter value